server/esi_config.py

```diff
@@ -4,6 +4,9 @@
 secret_key = 'XkkgTu89W1diw2OjrwGlav5T2koud7TPQoePbjoE'
 app_url = 'http://localhost:8080/'
 react_app_url = app_url + 'app/index.html'
+applicant_url = react_app_url + '?showing=applicant'
```

Comment:
Make sure to unit test these URLs for the right permissions down the line.

Reply:
they will all land in your `ensure_has_access` codes anyway

Reply:
You are not wrong, but without auth they see nothing of value at present. 

a) I'll make sure they never
b) I don't think this is alot more revealing that a path. I meant to use /applicant instead of ?showing=applicant, but routing in react if ugly sometimes